admin/proc-rhdh-deploy-eks-using-helm.adoc

```diff
@@ -0,0 +1,177 @@
+[id='proc-rhdh-deploy-eks-using-helm_{context}']
+= Deploying {product} in Elastic Kubernetes Services (EKS) using Helm Chart
+
+When you deploy {product-short} in Elastic Kubernetes Services (EKS) using Helm Chart, orchestrates a robust development environment within the AWS ecosystem.
+
+.Prerequisites
+
+* You have set the context to the EKS cluster in your current `kubeconfig`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html[Creating or updating a kubeconfig file for an Amazon EKS cluster].
+* You have installed `kubectl`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html[Installing or updating kubectl].
+* You have installed Helm 3 or latest. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/helm.html[Using Helm with Amazon EKS].
+* Your {product-short} application is running on EKS. For more information about using the AWS Application Load Balancer (ALB), see https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html[Application load balancing on Amazon EKS].
+* The ALB add-on is installed in the EKS cluster. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html[Installing the AWS Load Balancer Controller add-on].
+* You have subscribed to the registry.redhat.io. For more information, see https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+
+.Procedure 
+
+. Go to your terminal and run the following command to add the Helm chart repository containing the {product-short} chart to your local Helm registry:
++
+--
+[source]
+----
+$ helm repo add openshift-helm-charts https://charts.openshift.io/
+----
+--
+
+. Create a pull secret using the following command:
++
+--
+[source]
+----
+$ kubectl create secret docker-registry rhdh-pull-secret \
+    --docker-server=registry.redhat.io \
+    --docker-username=<user_name> \
+    --docker-password=<password> \
+    --docker-email=<email>
+----
+
+The created pull secret is used to pull the {product-short} images from the Red Hat Ecosystem. Also, add your username, password, and email address to the previous command.
+--
+
+. Create a file named `values.yaml` using the following template:
++
+--
+[source,yaml]
+----
+global:
+  # TODO Adjust this value if you know the DNS name under which
+  # your RHDH instance will be exposed.
+  # If using ALB with dynamic load balancer DNS, we'll be 
+  # able to adjust this value only after the ALB is provisioned.
+  host: rhdh.acme.com
```

Comment:
This section has been updated, and requires users to have their own domain, which they will set here. Please see the updated doc.

Reply:
Done